HTML/js/dodaj_skladnik.js

```diff
@@ -0,0 +1,79 @@
+window.onload = function(e){
+    console.log('onload')
+    }
+    
+    function validate() {
+        let err = 0;
+        if (!namevalidation('drinkname')) {
+            err++
+        }
+        if (!valuevalidation('drinvalue')) {
+            err++
+        }
+
+        if (err === 0 ){
```

Comment:
Expected indentation of 2 spaces but found 8.
Missing space before opening brace.